core/core-services-impl/src/test/java/com/sflpro/notifier/services/notification/NotificationProcessingServiceIntegrationTest.java

```diff
@@ -53,6 +53,7 @@ public void testProcessNotificationWithPushNotification() {
 
     }
 
+    @Ignore
```

Comment:
Aren't these tests supposed to be un-ignored?

Reply:
the first one depends on Amazon SNS service, so can't test; the second one is already fixed